pkg/cmd/step/helm/step_helm_apply.go

```diff
@@ -546,3 +500,75 @@ func (o *StepHelmApplyOptions) fetchSecretFilesFromVault(dir string, store confi
 	}
 	return files, nil
 }
+
+// Sets the Release name
+func setReleaseName(ns string, args []string, options *StepHelmApplyOptions) (releaseName string, err error) {
+
+	helmBinary, noTiller, helmTemplate, err := options.TeamHelmBin()
+	if err != nil {
+		return
+	}
+
+	_, devNs, err := options.KubeClientAndDevNamespace()
+	if err != nil {
+		return
+	}
+
+	if releaseName == "" {
+		if len(args) > 0 {
+			releaseName = args[0]
+		} else
+		if devNs == ns {
+			releaseName = platform.JenkinsXPlatformRelease
+		} else {
+			releaseName = ns
+			if helmBinary != "helm" || noTiller || helmTemplate {
+				releaseName = "jx"
+			}
+		}
+	}
+	return
```

Comment:
ditto

Reply:
I'll be logging error above, not sure what wrap up message would be needed here?